docker-compose.unixports.yml

```diff
@@ -0,0 +1,18 @@
+jenkins:
+    user: ${USER_ID}
+
+testintegration:
+    ports:
+        - "14064:14064"
+        - "14063:14063"
+omero:
+    ports:
+        - "4064:4064"
```

Comment:
`docker-compose.unixports.yml` is a default to run on isolation, `docker-compose.unix.yml` is to run on shared resources. You can choose by `EXTRAT=docker....yml ./ds`